### DISCUSSION TITLE
关于绳网等级

### DISCUSSION THREAD

**Discussion by zzyuJPG** [share121/inter-knot#3251] [其他]
由于绳网网站是基于Github的网站目前升级不能通过发表帖子、回复、写评论的方法提升，但是可以通过在个人的Github上创建库来提升等级，一个公共的库就会提升一级绳网等级（创建私人库没有效果），祝各位早日成为传奇绳匠<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3996afc68a13bfc5b5718af31e610f43_6807527097924316704.png" alt="罗刹 送花" width="75" >

**Comment by 1ctrl-cv** [comment]
传奇绳斤（

**Comment by aynuzbh** [comment]
无敌了
